tests/internals.rs

```diff
@@ -1,16 +1,26 @@
+use cairo_rs::vm::runners::cairo_runner::ExecutionResources;
 use felt::{felt_str, Felt};
 use lazy_static::lazy_static;
-use num_traits::Zero;
+use num_traits::{Num, Zero};
 use starknet_rs::{
     business_logic::{
+        execution::objects::{CallInfo, CallType, OrderedEvent, TransactionExecutionInfo},
         fact_state::{contract_state::ContractState, in_memory_state_reader::InMemoryStateReader},
         state::{cached_state::CachedState, state_api::StateReader},
+        transaction::objects::internal_invoke_function::InternalInvokeFunction,
     },
-    definitions::general_config::StarknetGeneralConfig,
-    services::api::contract_class::ContractClass,
-    utils::{felt_to_hash, Address},
+    definitions::{
+        constants::EXECUTE_ENTRY_POINT_SELECTOR,
+        general_config::{StarknetChainId, StarknetGeneralConfig},
+        transaction_type::TransactionType,
+    },
+    services::api::contract_class::{ContractClass, EntryPointType},
+    utils::{calculate_sn_keccak, felt_to_hash, Address},
+};
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
 };
-use std::{collections::HashMap, path::PathBuf};
 
 const ACCOUNT_CONTRACT_PATH: &str = "starknet_programs/account_without_validation.json";
 const ERC20_CONTRACT_PATH: &str = "starknet_programs/ERC20.json";
```

Comment:
Since you're using `starknet_programs/ERC20.cairo`, the file at `starknet_programs/blockifier/ERC20_without_some_syscalls/ERC20/ERC20.cairo` (which should be the same) is redundant and may be safely removed.